### PR TITLE
Reusable pipelines

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -1,28 +1,10 @@
...
-      - name: Upload coverage report
-        uses: actions/upload-artifact@v3
-        if: always()
-        with:
-          name: report
-          path: report.json
+  unit-tests:
+    uses: canonical/operator-workflows/.github/workflows/test.yaml@v1.0.0
+    secrets: inherit
+
```
Comment (mthaddon) [inline .github/workflows/test.yaml:10]: Can we remove this blank line?

Comment (arturo-seijas) [reply .github/workflows/test.yaml]: Done

**Review by mthaddon — APPROVED** [review]
Looks great! So nice to see so much code deletion :)

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Pull request merged**